Source/JavaScriptCore/runtime/ISO8601.cpp

```diff
@@ -886,7 +918,7 @@ static std::optional<CalendarRecord> parseCalendar(StringParsingBuffer<Character
     if (*buffer != ']')
         return std::nullopt;
     buffer.advance();
-    return CalendarRecord { isCritical, WTFMove(result) };
+    return CalendarRecord { isCritical, false, WTFMove(result) };
```

Comment:
Add /* isUnknown */ comment before the boolean to make it easy to understand what this boolean means